paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/ach/SyncViewModels.kt

```diff
@@ -0,0 +1,60 @@
+package com.stripe.android.paymentsheet.paymentdatacollection.ach
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.ui.platform.LocalContext
+import com.stripe.android.paymentsheet.ui.PrimaryButton
+import com.stripe.android.paymentsheet.viewmodels.BaseSheetViewModel
+import kotlinx.coroutines.flow.filterNot
+
+@Composable
+internal fun SyncViewModels(
+    viewModel: USBankAccountFormViewModel,
+    sheetViewModel: BaseSheetViewModel,
+) {
```

Comment:
Necessary for now to keep both view models in sync. In the future, we should have direct communication between them instead of via the UI layer.

Reply:
`:noddy-but-it-is-higher-rez-and-for-some-reason-that-makes-it-more-upsetting:`